### PR TITLE
CHE-1292. Add ability to divide Consoles area vertically and horizontally

### PR DESCRIPTION
**Pull request by RomanNikitenko**
consoles_split_horizontally
consoles_split_vertically

Signed-off-by: Roman Nikitenko rnikitenko@codenvy.com

### CODE REVIEW

**Comment by RomanNikitenko** [comment]
Add active/unactive state for tooltip buttons
active_state

**Comment by benoitf** [comment]
PR needs to be rebased

File: core/ide/che-core-ide-ui/src/main/java/org/eclipse/che/ide/ui/FontAwesome.java [hunk 1]
```diff
@@ -97,4 +97,19 @@
      */
     public static final String EXCLAMATION_TRIANGLE = "<i class=\"fa fa-exclamation-triangle\"></i>";
 
+    /**
+     * http://fontawesome.io/icon/columns/
+     */
+    public static final String COLUMNS = "<i class=\"fa fa-columns\" aria-hidden=\"true\"></i>";
```
Comment (vitaliy-guliy) [inline core/ide/che-core-ide-ui/src/main/java/org/eclipse/che/ide/ui/FontAwesome.java]: Remove aria-hidden=\"true\" and also from the following

File: plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java [hunk 2]
```diff
@@ -103,16 +101,17 @@
     private final CommandTypeRegistry          commandTypeRegistry;
     private final Map<String, ProcessTreeNode> machineNodes;
 
-    final List<ProcessTreeNode>                rootChildren;
-    final Map<String, TerminalPresenter>       terminals;
-    final Map<String, OutputConsole>           consoles;
-    final Map<OutputConsole, String>           consoleCommands;
+    final List<ProcessTreeNode>          rootChildren;
```
Comment (vitaliy-guliy) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java]: Formatting

File: core/ide/che-core-ide-ui/src/main/java/org/eclipse/che/ide/ui/FontAwesome.java (hunk 1 above)
Comment (RomanNikitenko) [inline core/ide/che-core-ide-ui/src/main/java/org/eclipse/che/ide/ui/FontAwesome.java]: thank you, I will fix it.

File: plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java (hunk 2 above)
Comment (RomanNikitenko) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java]: @vitaliy-guliy please see full changes - formatting is OK here.

File: core/ide/che-core-ide-ui/src/main/java/org/eclipse/che/ide/ui/FontAwesome.java (hunk 1 above)
Comment (RomanNikitenko) [inline core/ide/che-core-ide-ui/src/main/java/org/eclipse/che/ide/ui/FontAwesome.java]: @vitaliy-guliy Fixed!

**Comment by vparfonov** [comment]
ok

**Pull request merged**